compiler/noirc_frontend/src/noir_parser.lalrpop

```diff
@@ -4,7 +4,7 @@ use crate::lexer::token::BorrowedToken;
 use crate::lexer::token as noir_token;
```

Comment:
Should this file be kept up to date? It seems to have only a subset of the grammar. I made the changes to make it compile, I don't know if that's good enough.

Reply:
It doesn't need to be. This work has been shelved for now and the plan is to revisit it post 1.0.